testsuite/podman_runner/09_build_server_code.sh

```diff
@@ -7,7 +7,9 @@ sudo -i podman exec uyuni-server-all-in-one-test bash -c "[ -d /usr/share/susema
 sudo -i podman exec uyuni-server-all-in-one-test bash -c "[ -d /usr/share/susemanager/www/htdocs ] || mkdir -p /usr/share/susemanager/www/htdocs"
 sudo -i podman exec uyuni-server-all-in-one-test bash -c "[ -d /usr/share/susemanager/www/tomcat/webapps ] || mkdir -p /usr/share/susemanager/www/tomcat/webapps"
 
-sudo -i podman exec uyuni-server-all-in-one-test bash -c "cd /java && ant -f manager-build.xml ivy refresh-branding-jar deploy-local"
+# If ivy build fails, try again, because 50% of times fails when downloading the new jar files from the open build service
+sudo -i podman exec uyuni-server-all-in-one-test bash -c "cd /java && ant -f manager-build.xml ivy || ant -f manager-build.xml ivy || ant -f manager-build.xml ivy"
+sudo -i podman exec uyuni-server-all-in-one-test bash -c "cd /java && ant -f manager-build.xml refresh-branding-jar deploy-local"
```

Comment:
Wouldn't it make sense to wrap e.g. the `ant` command in a do-while loop and check the exit code for it?